includes/course-theme/class-sensei-course-theme-quiz.php

```diff
@@ -120,12 +120,22 @@ private function maybe_add_quiz_results_notice() {
 		} elseif ( 'failed' === $quiz_progress->get_status() ) {
 			$passmark         = get_post_meta( $quiz_id, '_quiz_passmark', true );
 			$passmark_rounded = Sensei_Utils::round( $passmark, 2 );
-			$text             = sprintf(
-				// translators: The first placeholder is the minimum grade required, and the second placeholder is the actual grade.
-				__( 'You require %1$s%% to pass this quiz. Your grade is %2$s%%.', 'sensei-lms' ),
-				$passmark_rounded,
-				$grade_rounded
-			);
+
+			if ( get_locale() === 'en_US' || ( function_exists( 'has_translation' ) && has_translation( 'You must score at least %1$s%% to pass this quiz. Your grade is %2$s%%.', 'sensei-lms' ) ) ) {
```

Comment:
Can we extract these three checks to a helper function? 🤔

Reply:
For sure! Updated in these 2 commits:

- https://github.com/Automattic/sensei/pull/7793/commits/996323eb259f185dd25da528e37dcdaf0f8b5a0c
- https://github.com/Automattic/sensei/pull/7793/commits/83314eafdaa253ddd6292de7f1a0abac5c70d447